libs/crypto/keys_manager_impl.hpp

```diff
@@ -26,8 +26,10 @@ namespace iroha {
   class KeysManagerImpl : public KeysManager {
    public:
     explicit KeysManagerImpl(std::string account_name);
+    KeysManagerImpl();
```

Comment:
May you explain why this is here?

Reply:
This code is temporal, key manager should not be in peer at all. 
This is just a quick workaround to load keys to be able to actually run iroha